src/js/background/sync.js

```diff
@@ -585,3 +603,21 @@ async function setAssignmentWithUUID(assignedSite, urlKey) {
   }
   throw new Error (`No cookieStoreId found for: ${uuid}, ${urlKey}`);
 }
+
+async function getExcludeRegExp() {
+  const { syncExcludeRegExp } = await browser.storage.local.get("syncExcludeRegExp");
+  if (syncExcludeRegExp) {
+    return new RegExp(syncExcludeRegExp, "i");
+  } else {
+    return false;
+  }
+}
+
+function excludeRegExpMatchesIdentity(excludeRegExp, identity) {
+  if (excludeRegExp && identity.name.match(excludeRegExp)) {
+    if (SYNC_DEBUG) console.log(`Exclude regexp matches identity '${identity.name}'`);
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
I know we aren't doing it for the rest of the code put I think it'd be great to have a short description and what's the variable passed and spitted. It could save someone else (maybe future you or me) time and maybe headaches :).

Reply:
Comments added. Never too late to start documenting things. :+1: